bigframes/_config/__init__.py

```diff
@@ -95,12 +105,21 @@ def sampling(self) -> sampling_options.SamplingOptions:
         (e.g., to_pandas, to_numpy, values) or implicitly (e.g.,
         matplotlib plotting). This option can be overriden by
         parameters in specific functions.
+
+        Returns:
+            bigframes._config.sampling_options.SamplingOptions:
+              Options for controlling downsampling.
```

Comment:
Nit: Should be indented another two spaces. We use multiples of 4 spaces for indentation.

```suggestion
                Options for controlling downsampling.
```